### DISCUSSION TITLE
How to update custom interface when collection form edit and save

### DISCUSSION THREAD

**Discussion by sandyksit** [directus/directus#15142] [Archive]
@rijkvanzanten ,

How to update custom interface when other collection form edit and save?  I have used updated method (vue lifecyle) in interface vue file. I need to display message in interface when collection form edit (save and stay), 
Note: previously it was working but now its not working

Thanks,
Sandeep

**Comment by JonathanSchndr** [comment]
> @rijkvanzanten ,
> 
> How to update custom interface when other collection form edit and save? I have used updated method (vue lifecyle) in interface vue file. I need to display message in interface when collection form edit (save and stay), Note: previously it was working but now its not working
> 
> Thanks, Sandeep

Hey sandyksit, I'm not Rijk, but I'm happy to try to help you too. I think you are using the custom interface wrong right now. You don't have to worry about saving or editing the interface. This is just the interface to create a new type of interface. Feel free to look here again: https://docs.directus.io/extensions/interfaces.html

**Comment by rijkvanzanten** [comment]
@sandyksit Interfaces don't currently get a message or event coming in when the parent form saves. What did you try before that no longer works?

---

ps no need to tag me directly, I have notifications enabled for everything on the repo 😉

**Reply by sandyksit** [reply]
I am trying to save the collection form, but I have 'preview ' and 'promote' button in one custom interface, on save and stay I need to send some Boolean  notification to my custom interface for activate the buttons.

**Reply by rijkvanzanten** [reply]
Hmm there's no way to listen to that save and stay event. That being said, you could maybe check if the form has edits? If the item has a primary key, and it has no edits, it would mean that the item was previously saved 🤔
